spec.html

```diff
@@ -2805,32 +2805,64 @@
-        </li>
-        <li>
-          <dfn aoid="IsUnresolvableReference" id="ao-isunresolvablereference">IsUnresolvableReference</dfn>(_V_). Returns *true* if the base value component of the reference _V_ is *undefined*; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsSuperReference" id="ao-issuperreference">IsSuperReference</dfn>(_V_). Returns *true* if the reference _V_ has a thisValue component; otherwise returns *false*.
-        </li>
-      </ul>
       <p>The following abstract operations are used in this specification to operate on references:</p>
 
+      <emu-clause id="sec-getbase" aoid="GetBase">
+        <h1 id="ao-getbase">GetBase ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the base value component of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-getreferencedname" aoid="GetReferencedName">
+        <h1 id="ao-getreferencedname">GetReferencedName ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the referenced name component of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-isstrictreference" aoid="IsStrictReference">
+        <h1 id="ao-isstrictreference">IsStrictReference ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the strict reference flag of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-hasprimitivebase" aoid="HasPrimitiveBase">
+        <h1 id="ao-hasprimitivebase">HasPrimitiveBase ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. If Type(_V_'s base value component) is Boolean, String, Symbol, or Number, return *true*; otherwise return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-ispropertyreference" aoid="IsPropertyReference">
+        <h1 id="ao-ispropertyreference">IsPropertyReference ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. If either the base value component of _V_ is an Object or HasPrimitiveBase(_V_) is *true*, return *true*; otherwise return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-isunresolvablereference" aoid="IsUnresolvableReference">
+        <h1 id="ao-isunresolvablereference">IsUnresolvableReference ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. If the base value component of _V_ is *undefined*, return *true*; otherwise return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-issuperreference" aoid="IsSuperReference">
+        <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. If _V_ has a thisValue component, return *true*; otherwise return *false*.
```

Comment:
I wouldn't think so: it's just the name of a component of the Reference value.

More generally, there's the point that a Reference is modelled as an ad hoc thing-with-components, rather than as a record, but that's not the point of this PR.